pkg/codegen/schema_test.go

```diff
@@ -174,7 +174,7 @@ func TestProperty_GoTypeDef(t *testing.T) {
 		// When field is write only, it will always be pointer unless pointer is
 		// skipped by setting SkipOptionalPointer flag
 		{
-			name: "When field is write only",
+			name: "When field is write only and read only pointer disabled",
```

Comment:
is the name of this test right, when the flag is `GlobalStateDisableRequiredReadOnlyAsPointer: true`? Or should we rename the tests?

Reply:
I did not understand what you mean. 
Are you saying rename to the following: 
```
"When field is write only and the flag is GlobalStateDisableRequiredReadOnlyAsPointer: true
```

What is incorrect here?

Reply:
Ignore me, I'd misread